core/src/main/java/org/apache/accumulo/core/spi/file/rfile/compression/None.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.spi.file.rfile.compression;
+
+import org.apache.accumulo.core.file.rfile.bcfile.IdentityCodec;
+
+import com.google.auto.service.AutoService;
+
+@AutoService(CompressionAlgorithm.class)
+public class None implements CompressionAlgorithm {
```

Comment:
I would prefer a more descriptive class name. You could call it `NoCompression`. This would be consistent with how we named the `NoCryptoSerivce`

Reply:
I rename `None` to `NoCompression` in 9b22ead